apps/web/src/lib/api/getMarkup.tsx

```diff
@@ -0,0 +1,171 @@
+import type { MediaSet, NftImage, Profile } from '@generated/types';
+import getIPFSLink from '@lib/getIPFSLink';
+
+export const markUp = (profile: Profile & { picture: MediaSet & NftImage }) => {
+  const name = profile?.name;
+  const handle = `@${profile?.handle}`;
+  const description = profile?.bio ?? '';
+  const image = profile
+    ? `https://ik.imagekit.io/lensterimg/tr:n-avatar/${getIPFSLink(
+        profile?.picture?.original?.url ??
+          profile?.picture?.uri ??
+          `https://avatar.tobi.sh/${profile?.ownedBy}_${profile?.handle}.png`
+      )}`
+    : 'https://assets.lenster.xyz/images/og/logo.jpeg';
+
+  const { totalFollowing, totalFollowers } = profile.stats;
+  return (
```

Comment:
`tw` didn't worked 😕

Reply:
done 🖖🏻

Reply:
@bigint  move `nFormatter` to common place 😮‍💨 and see preview